params.yaml

```diff
@@ -359,6 +359,10 @@ ratio_study:
   near_stage: "certified"
   near_column: "meta_certified_tot"
 
+  # Min. number of sales to calculate ratio statistics, per CCAO SOPs. Groups
+  # must have greater than or equal to the number of sales specified
+  min_n_sales: 30
```

Comment:
This is the minimum number of sales required to calculate a ratio statistic per group/geography. It's lowered to 30 (from 33) because previously the `ccao_` ratio stat functions would trim the top and bottom 5% of sales per IAAO recommendations (so a min of 33 would result in ~30 sales).